### ISSUE TITLE
Mapbox "Uncaught ReferenceError: e is not defined"

### ISSUE THREAD

**Issue by jvandenaardweg** [kaliberjs/build#135]
We got a `Uncaught ReferenceError: e is not defined` on a clients test environment that uses a React version of Mapbox GL. After digging in that error, there's a possible solution in the Uglify options in Webpack: https://github.com/alex3165/react-mapbox-gl/issues/200#issuecomment-370175270

`typeofs: false` is the fix

`/build/lib/build.js`

```javascript
new UglifyJsPlugin({
  cache: true,
  parallel: true,
  sourceMap: true,
  uglifyOptions: {
    compress: {
      typeofs: false /* added so the build works with Mapbox: https://github.com/alex3165/react-mapbox-gl/issues/200#issuecomment-370175270 */
    }
  }
})
```
What `typeofs: false` does:
> typeofs (default: true) -- Transforms typeof foo == "undefined" into foo === void 0. Note: recommend to set this value to false for IE10 and earlier versions due to known issues.

Tested locally and issue is fixed with this. But we need this in the build asap so our test environment can use this :)

**Comment by EECOLOR** [comment]
`comparisons: false` might be an option as well

According to this issue the problem should be solved: https://github.com/mapbox/mapbox-gl-js/issues/4359#issuecomment-303880888

This is released in version 0.47

**Comment by jvandenaardweg** [comment]
It seems the used library released a newer version (3.3.2) on NPM (not in their GitHub repo) which fixes the Mapbox problem. So we'll use that: https://www.npmjs.com/package/react-map-gl

I'll close this issue, since we resolved it that way.

Thanks for the tip!
